db/sql/test_0_msar.sql

```diff
@@ -1378,7 +1378,7 @@ $f$ LANGUAGE plpgsql;
 
 CREATE OR REPLACE FUNCTION test_add_mathesar_table_badname() RETURNS SETOF TEXT AS $f$
 DECLARE
-  badname text := $b$M"new"'dsf' \t"$b$;
+  badname text := '"new"''dsf'' \t"';
```

Comment:
Minor issue: This isn't the same string. Regardless, it still tests the same thing. I just don't like having to think while reviewing.